src/app/(proper_react)/redesign/(public)/LandingView.tsx

```diff
@@ -42,37 +43,79 @@ export const View = (props: Props) => {
             isHero
             eligibleForPremium={props.eligibleForPremium}
             signUpCallbackUrl={`${process.env.SERVER_URL}/redesign/user/dashboard/`}
+            eventId={{
+              cta: "clicked_get_scan_header",
+              field: "entered_email_address_header",
+            }}
           />
         </div>
         <div className={styles.heroImage}>
           <HeroImage {...props} />
         </div>
       </header>
 
-      <div className={styles.valuePropositionWrapper}>
+      <section className={styles.quoteWrapper}>
+        <div className={styles.quote}>
+          <h2>
```

Comment:
Since this isn't a heading accompanying some content, I think a `<b>` is probably more appropriate than an `<h2>`. (This is relevant because some people navigate pages by first pulling up a list of all heading elements, and then jumping to the relevant part of the page - and this one would just be noise.)